docs/Axis_Twist_Compensation.md

```diff
@@ -25,31 +25,31 @@ try to probe the bed without attaching the probe if you use it.
 > correctly set as they greatly influence calibration.
 
 ### Basic Usage: X-Axis Calibration
-1. After setting up the ```[axis_twist_compensation]``` module, run:
+1. After setting up the `[axis_twist_compensation]` module, run:
 ```
 AXIS_TWIST_COMPENSATION_CALIBRATE
 ```
 This command will calibrate the X-axis by default.
-    - The calibration wizard will prompt you to measure the probe Z offset at
+  - The calibration wizard will prompt you to measure the probe Z offset at
     several points along the bed.
-    - By default, the calibration uses 3 points, but you can specify a different
+  - By default, the calibration uses 3 points, but you can specify a different
     number with the option:
 ``
 SAMPLE_COUNT=<value>
 ``
 
 2. **Adjust Your Z Offset:**
-After completing the calibration, be sure to [adjust your Z offset]
-(Probe_Calibrate.md#calibrating-probe-z-offset).
+After completing the calibration, be sure to
+[adjust your Z offset](Probe_Calibrate.md#calibrating-probe-z-offset).
 
-3. **Perform Bed Leveling Operations:**
+4. **Perform Bed Leveling Operations:**
```

Comment:
Thanks.  Why was the numbering changed from 3 to 4 here?

Otherwise looks fine to me.
-Kevin

Reply:
thanks for the review @KevinOConnor, I fixed the numbering.